src/media/js/site/containers/landing.js

```diff
@@ -1,17 +1,65 @@
 import React from 'react';
+import {connect} from 'react-redux';
 import {ReverseLink} from 'react-router-reverse';
+import {history} from 'react-router/lib/BrowserHistory';
+import {bindActionCreators} from 'redux';
 
-import {Page} from '../../site/components/page';
+import {fxaLoginBegin, login} from '../actions/login';
 
+import {LoginButton} from '../components/login';
+import {Page} from '../components/page';
+
+
+class Landing extends React.Component {
+  loginHandler = authCode => {
+    // Call login, passing in some extra stuff from siteConfig.
+    this.props.login(authCode, this.props.siteConfig.authState,
+                     this.props.siteConfig.clientId);
+  }
 
-export default class Landing extends React.Component {
   render() {
     return (
-      <Page title="Develop HTML5 content for an open marketplace.">
-        <ul>
-          <li><ReverseLink to="addon">Firefox OS Add-ons</ReverseLink></li>
-        </ul>
+      <Page className="landing">
+        <header className="landing--header">
```

Comment:
i'd just use a div, we have a header element already and this doesn't really look like a header